packages/react-components/react-divider/src/stories/Divider/DividerAlignContent.stories.tsx

```diff
@@ -1,7 +1,5 @@
 import * as React from 'react';
-import { makeStyles } from '@griffel/react';
-import { tokens } from '@fluentui/react-theme';
-import { Divider } from '@fluentui/react-divider';
+import { makeStyles, tokens, Divider } from '@fluentui/react-components';
```

Comment:
it'd be nice to move Divider to the beginning to alphabetize these too

Reply:
we've actually been following a lowercase to uppercase sorting order for all of these 😅